docs/concepts/cluster-administration/certificates.md

```diff
@@ -8,26 +8,7 @@ title: Certificates
 ## Creating Certificates
 
 When using client certificate authentication, you can generate certificates
-using an existing deployment script or manually through `easyrsa`, `openssl`
-or `cfssl`.
-
-### Using an Existing Deployment Script
```

Comment:
Is this information still relevant? It doesn't seem to be related to removing saltbase from the docs.

Reply:
Yes, `using an existing deployment script` related to it, so remove, and `manually through easyrsa, openssl or cfssl`  keep maintain.